include/box2d/b2_collision.h

```diff
@@ -23,8 +23,10 @@
 #ifndef B2_COLLISION_H
 #define B2_COLLISION_H
 
+#include <climits>
```

Comment:
is this needed? Doesn't seem related to the current PR, right?

Reply:
It's a bonus, headers in `.h` have been deprecated for years. They may not be available in a standard compiler (even if the may compilers make them available). It's a good practice to use C++ version of C headers.

Reply:
A few years ago I had some issues with box2d because it was using some of these "modern" headers (on a proprietary platform). Changing that might break compilation on some platforms and doesn't add much anyway.

Reply:
Non-conformant compilers should die.